### ISSUE TITLE
Tracking issue: Allow to quote message in reply from newsroom

### ISSUE THREAD

**Issue by mattwr18** [tactilenews/100eyes#1478]
```
As an editor
I would like to be able to reply to a message even after I have sent out another request, but this can be confusing to the contributor because they don't know what message I am replying to. I would like to be able to quote their message in my reply, as I would do on an actual messenger.
So that I can reply when I have time and it's not confusing for the contributor
```

**Comment by mattwr18** [comment]
As of [signal-cli 0.10.0](https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#0100---2021-12-11), we can send a quoted message with Signal. This feature has not yet been added to `signal-cli-rest-api`, but there is an open PR [ Added support for missing send arguments #208](https://github.com/bbernhard/signal-cli-rest-api/pull/208) that has some traction

**Comment by mattwr18** [comment]
With Telegram, this is already supported. `sendMessage` accepts an optional `reply_to_message_id`, which can be used to reply to that message. See [Telegram docs for sendMessage](https://core.telegram.org/bots/api#sendmessage)

See attached screenshot :point_down:  where I got it working locally.

Note that we would need to add an attribute to the `Message` model to save the original Telegram `message_id`.

![Screenshot from 2022-09-29 13-10-40](https://user-images.githubusercontent.com/26943915/193016354-a0865600-0c32-405a-a340-e74aef9f62e8.png)

**Comment by mattwr18** [comment]
For Threema, there does not seem to be anything in the docs that suggest that one can reply to a message by referencing the messageId. We do receive a `messageId`, which we could save, but there doesn't appear to be a way to reference it when we send a message.

I have checked the threema-forum.de and translated it to English, but have not seen anything there. Maybe, we could reach out to support to ask them about this and the profile image issue @drjakob ?

**Comment by mattwr18** [comment]
I have sent an email to Threema support about this and another issue and will update here with their response.

**Comment by mattwr18** [comment]
Threema got back to me and said:

> Dear Mr. Rider,
> 
> Thank you for your patience. I am glad to answer your questions as follows.
> 
> It is possible to quote a message in Threema Gateway. However, this feature isn't
> supported in the SDKs. If you would like to quote a message, please do as follows (without
> …):
> 
> …
> 
> `> quote #00112233ccddeeff`
> 
> This is the comment
> 
> …
> 
> In this case, «00112233ccddeeff» is the message id, which you need to quote.

I have tested this and it works!

For `signal-cli-rest-api` the [creator of the PR I linked to said](https://github.com/bbernhard/signal-cli-rest-api/pull/208#issuecomment-1282517818):

> I'll try to wrap it up this week

Maybe, we could start working on this issue even.

**Comment by mattwr18** [comment]
@drjakob @FrauCsu, the `signal-cli-rest-api` implementation of this has been merged and is waiting to be officially released. We are pretty close to being able to start working on this, if we want to prioritize it. I guess we would not start on it in this quarter though, right?
[Added support for missing send arguments #208](https://github.com/bbernhard/signal-cli-rest-api/pull/208)

**Comment by mattwr18** [comment]
Moving this to a future quarter as we haven't even started working on it. 

Before starting, we might want to wait till the implementation of signal-cli-rest-api is released and decide on some design questions on how to display quoted messages in the UI in our app.

**Comment by mattwr18** [comment]
I guess this would be ready to be worked on if we wanted to prioritize it @drjakob @FrauCsu ...
